htlcswitch/link.go

```diff
@@ -493,7 +493,7 @@ var _ ChannelLink = (*channelLink)(nil)
 // NOTE: Part of the ChannelLink interface.
 func (l *channelLink) Start() error {
 	if !atomic.CompareAndSwapInt32(&l.started, 0, 1) {
-		err := errors.Errorf("channel link(%v): already started", l)
+		err := fmt.Errorf("channel link(%v): already started", l)
```

Comment:
seems like this commit can be cherry-picked out of this PR and be merged into master first.

Reply:
I have put it at the beginning of the PR series and will cut a new branch and PR it in on its own.